diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter5/ParameterNameInOuterLambdaRule.kt

```diff
@@ -24,9 +29,19 @@ class ParameterNameInOuterLambdaRule(configRules: List<RulesConfig>) : DiktatRul
     }
 
     private fun checkLambda(node: ASTNode) {
-        val hasInnerLambda = node
-            .findAllDescendantsWithSpecificType(KtNodeTypes.LAMBDA_EXPRESSION, false)
-            .isNotEmpty()
+        val configuration = ParameterNameInOuterLambdaConfiguration(
+            configRules.getRuleConfig(PARAMETER_NAME_IN_OUTER_LAMBDA)?.configuration
+                ?: emptyMap()
```

Comment:
Maybe it's better to extract this from method?

Reply:
Done